content/en/blog/_posts/2021-06-21-update-with-ingress-nginx.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: blog
+title: 'Updates to Ingress Nginx with network v1'
+date: 2021-06-21
+slug: update-with-ingress-nginx
+---
+
+**Authors:** James Strong, Richardo Katz 
+
+With all Kubernetes APIs, there is a process to creating, maintaining, and
+ultimately deprecating them once they become GA. The networking.k8s.io API is no
+different. With the release of Kubernetes 1.22, The networking.k8s.
+io/v1beta1 API version of IngressClass will no longer be served in 
+Kubernetes v1.22. The extensions/v1beta1 and networking.k8s.io/v1beta1 API 
+versions of Ingress also will no longer be served in v1.22.  This move has been 
+in discussion since
+[2017](https://github.com/kubernetes/kubernetes/issues/43214),
+[2019](https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/) with 
+1.16 Kubernetes API deprecations,and most recently in 
+KEP-1453: 
+[Graduate Ingress API to GA](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/1453-ingress-api#122).
+
+During our community meetings, the group has decided to continue 
+supporting Kubernetes versions older than 1.22 with Ingress version 0.47.0. 
```

Comment:
There is no v0.47.0 version of the Ingress API. Did you mean Ingress-NGINX instead?

Reply:
yes, updated to clarify that point